native/src/main/java/io/ballerina/lib/redis/config/ConnectionParams.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org)
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.lib.redis.config;
+
+/**
+ * Represents connection parameters based configurations.
+ *
+ * @param host                host name
+ * @param port                port number
+ * @param userName            username of the connection
```

Comment:
Can't we use `username` here?

Reply:
Changed with https://github.com/ballerina-platform/module-ballerinax-redis/pull/218/commits/dc84a3674a3719a4f1c5038643c414f56715c468